src/System.Runtime/tests/System/Reflection/AssemblyTests.cs

```diff
@@ -32,6 +32,8 @@ public class AssemblyTests
 
         static AssemblyTests()
         {
+            loadFromTestPath = null;
+#if !uapaot  // Assembly.Location not supported (properly) on uapaot.
```

Comment:
Doesn't look temporary anymore - winds seem to be leaning towards returning the empty string here. In which case, I'd consider it "not supported" even though we're returning just enough to avoid a NullRefException.